inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -462,7 +463,7 @@ std::shared_ptr<ngraph::Node> V10Parser::createNode(const std::vector<ngraph::Ou
 
     // Check that operation in default opsets
     auto isDefaultOpSet = [](const std::string& version) -> bool {
-        for (size_t i = 1; i <= 5; i++) {
+        for (size_t i = 1; i <= 6; i++) {
```

Comment:
can we extract this number as `opsets.size()`?

Reply:
No, we cannot because opsets can also contains custom operation sets. 
In any case it is a legacy code and should be removed soon.